identity/handler.go

```diff
@@ -53,7 +56,7 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 type identityResponse struct {
 	// required: true
 	// in: body
-	Body *Identity
+	Body *IdentityRepresentation
```

Comment:
Instead of creating a full, new model why not a composite struct?

```suggestion
	Body struct {
	  Identity
	  // any other field like:
	  Foo string `json:"foo"`
	}
```

Reply:
Thanks for your feedback @aeneasr !
This is what I initially tried, but I could not get this to survive the go-swagger cycle. For me it looks like it's not possible to reproduce the embedded Type in the generated client code.

Reply:
Ah, damn, that is possibly a problem. An alternative would also be to supply an openapi patch: https://github.com/ory/kratos/tree/master/.schema/openapi/patches

Reply:
I wasn't aware of the patches and will have a look. Thank you!

Reply:
@aeneasr I made it work with an embedded struct and also switched Naming to lower case. Some remarks:
 - I had to remove type ( ...) in identity with single lines since go-swagger generates different output in this case
 - The e2e test fail in circle-ci and I don't quite understand why. I am not sure this is related to my changes. Could you take  a look?

Reply:
Sorry for the long delay, will take a look now!